cips/cip-29.md

```diff
@@ -0,0 +1,199 @@
+| **9**   | 2.77           | 7.91             | 5.54             | 4.26             |
+| **10**  | 2.58           | 7.37             | 5.16             | 3.97             |
+| **11**  | 2.41           | 6.89             | 4.82             | 3.71             |
+| **12**  | 2.25           | 6.43             | 4.5              | 3.46             |
+| **13**  | 2.1            | 6.0              | 4.2              | 3.23             |
+| **14**  | 1.96           | 5.6              | 3.92             | 3.02             |
+| **15**  | 1.83           | 5.23             | 3.66             | 2.82             |
+| **16**  | 1.7            | 4.86             | 3.4              | 2.62             |
+| **17**  | 1.59           | 4.54             | 3.18             | 2.45             |
+| **18**  | 1.5            | 4.29             | 3.0              | 2.31             |
+
+## **Backwards Compatibility**
+
+No backward compatibility issues are anticipated. This parameter change follows the standard CIP process, and current node operators should not require any additional changes besides upgrading to the new binary which automatically implements the changes discussed in this CIP.
+
+## **Test Cases**
+
+1. Parameter Verification Test:
+
+   * Ensure that when the updated inflation parameters are applied at Year 3, the on-chain inflation rate reflects the new values instead of the old schedule.
+
+2. Reward Distribution Test:
+
+   * Validate that block rewards are distributed correctly under the new inflation rates.
+
+## **Reference Implementation**
+
+Below is a pseudo-code snippet for updating the inflation parameters. Actual implementation may vary by client:
+
+Proposed implementation:
+
+```go
+func (m Minter) CalculateInflationRate(ctx sdk.Context, genesis time.Time) sdk.Dec {
+	years := yearsSinceGenesis(genesis, ctx.BlockTime())
+
+	// Default inflation calculation
+	inflationRate := InitialInflationRateAsDec().Mul(
+		sdk.OneDec().Sub(DisinflationRateAsDec()).Power(uint64(years)),
+	)
+
+	// For AppVersion \> 3, adjust the inflation rate:
+	if ctx.ConsensusParams().Version.AppVersion \> 3 {
+		// First, reduce the current inflation rate by 33%
+		inflationRate \= inflationRate.Mul(sdk.NewDecWithPrec(67, 2)) // 0.67 \= 67%
+
+		// Then, if we are in year two or later, apply a one-time disinflation of 6.7%
+		if years \>= 2 {
+			inflationRate \= inflationRate.Mul(sdk.OneDec().Sub(sdk.NewDecWithPrec(67, 3))) // 1 \- 0.067 \= 0.933
+		}
+	}
+
+	// Ensure the inflation rate does not fall below the target inflation rate.
+	if inflationRate.LT(TargetInflationRateAsDec()) {
+		return TargetInflationRateAsDec()
+	}
+
+	return inflationRate
+}
+```
+
+## **Alternative Approaches**
+
+Alternatives that were considered.
+
+### **Alternative 0: Do nothing**
+
+If we continue to do nothing, Celestia’s inflation schedule may not be competitive with other networks’ and TIA’s onchain usage may be limited.
```

Comment:
This is phrased confusingly b/c the inflation schedule doesn't need to be competitive with other networks'. The yield earned from staking _may_ want to be competitive with other networks but if the inflation remains high then the yield earned from staking will be competitive. This proposal is advocating for reducing the inflation rate so that would reduce the staking yield which makes it less competitive with other networks'.

Can you rephrase if that's not what is meant here?